### PR TITLE
Updating AMD Representation

### PR DESCRIPTION
**Pull request by larrydewey**
Adding @ryansavino as the second representative for AMD.

### CODE REVIEW

File: MAINTAINERS
Comment (fitzthum) [inline MAINTAINERS:12]: IT is not necessary to update this file. We might actually want to delete it to avoid confusion.

**Review by fitzthum — APPROVED** [review]
See note

**Comment by fidencio** [comment]
So, this raises what criteria we have for having one, two, or more folks representing a company as part of the SC.

What I've imagined so far was having different parts of the same company working on the project, but that was my interpretation only, as I don't recall it documented anywhere (but maybe it is, and I'd appreciate if we have a link for this).

I'd like to have this clarified first before adding Ryan, as this opens the door for having the different parts of Intel being better represented at the SC as well.

**Comment by fidencio** [comment]
> So, this raises what criteria we have for having one, two, or more folks representing a company as part of the SC.
> 
> What I've imagined so far was having different parts of the same company working on the project, but that was my interpretation only, as I don't recall it documented anywhere (but maybe it is, and I'd appreciate if we have a link for this).
> 
> I'd like to have this clarified first before adding Ryan, as this opens the door for having the different parts of Intel being better represented at the SC as well.

Let me reply to myself here.

From https://github.com/confidential-containers/confidential-containers/blob/main/governance.md#selection, we can see:
"""
The convention of the SC is for each organization that is significantly engaged in the project to be represented by one or two members. In addition to a company, an organization could also refer to a non-corporate institution such as school. Selection of the representatives of each organization is not within scope of the SC. The organization itself should select the individual(s) to represent it.
"""

By Tobin's reply, I think that this is being treated as an expansion (as it adds one more member).
"""
Expansion
The SC can be expanded if a new organization begins making significant contributions to the project. When evaluating requests for expansion, the SC will mainly consider whether the organization in question is making significant contributions to the community. There is no standard definition for significant contributions. The SC should prioritize including relevant stakeholders. The candidates from a organization are understood to represent that organization. The SC may do some basic checks to ensure that candidates are familiar with the project and represent relevant contributors. Expansion should be approved by at least 2/3rds of current SC members.
"""

However, personally, I don't see this as an expansion, but rather a company nominating another person, which falls more into the following sentence: "The organization itself should select the individual(s) to represent it.".

I'd say it's up for interpretation, but we should clarify it.

With everything said and quoted, I think if an organisation wants to add another member, it should be up to the organisation to do so, not requiring the 2/3 of the votes to have it approved, but that's my 2 cents.

All in all, lgtm!

**Comment by fitzthum** [comment]
I agree that things are a bit ambiguous. I went for expansion because the charter says that `Membership changes can fall into one of four categories`. This proposal clearly does not fall into the other three categories.

That said, the expansion protocol should be more precise. For instance it says that `The SC can be expanded if a new organization begins making significant contributions to the project.` Since AMD is not a new organization this proposal doesn't quite fit under expansion.

Looking more at the spirit of the protocol it is true that the steering committee tries to leave selection of individual members to each organization, but I think it is within the scope of the SC to approve the number of members that an organization has on the SC. After all, when a new organization comes to the SC, the SC approves the number of members they will have based on the contributions of the organization. I think it's reasonable to follow the same protocol when moving from 1 to 2 members. Although I think we should do evaluate these types of proposals fairly liberally.

Given AMD's level of contribution and the nature of Larry's upcoming absence, it doesn't seem ideal to go through the such a complex protocol, but this still looks like an expansion to me. I guess Larry change his PR and opt for replacement instead, but it might be better to get this sorted out.

I don't see potential discussion about expanding beyond two members for one company as related to this issue.

@larrydewey please add the DCO to your commits.

**Comment by fidencio** [comment]
> I agree that things are a bit ambiguous. I went for expansion because the charter says that Membership changes can fall into one of four categories. This proposal clearly does not fall into the other three categories.
> 
> That said, the expansion protocol should be more precise. For instance it says that The SC can be expanded if a new organization begins making significant contributions to the project. Since AMD is not a new organization this proposal doesn't quite fit under expansion.

I agree, and later on we can discuss and propose a change for that.  For now, we can make things simpler by doing a "replacement" and a "replacement" again once Larry is back, and by that time I hope things will be sorted out.

> Given AMD's high level of contribution

So, this is the part that we should clarify, IMHO.
I"m **not** saying AMD doesn't provide high level of contribution, but we have to have a clear message to whoever already joined, or will join, about this.

> and the nature of Larry's upcoming absence, it doesn't seem ideal to go through such a complex protocol, but this still looks like an expansion to me.

This, IMHO, calls for a replacement rather than an expansion, mainly due to the nature of the absence, and you mentioned that Larry will change his PR for that, and that will be a no-brainer / no-discussion kind of approval.

> I guess Larry change his PR and opt for replacement instead, but it might be better to get this sorted out.

I'll mark this as "changes requested" to have this reflected in the PR.

> I don't see potential discussion about expanding beyond two members for one company as related to this issue.

Me neither, I brought this up and immediately corrected myself in the next comment. :-)



Please, don't take me wrong on this one, I'm trying to be as fair as possible to avoid situations where we open a can of worms for ourselves to have to deal in the future.

**Review by fidencio — CHANGES_REQUESTED** [review]
As mentioned by Tobin in one of the comments on this PR:
"I guess Larry change his PR and opt for replacement instead, but it might be better to get this sorted out."

I'm marking this as "Request changes" till that happens.

Thanks @larrydewey!

File: MAINTAINERS
Comment (larrydewey) [reply MAINTAINERS]: Woops. Thanks @fitzthum. Removed commit.

**Comment by larrydewey** [comment]
@fidencio Although it may take longer, I believe I will opt for an expansion vote.

**Comment by fidencio** [comment]
@larrydewey, makes sense.

I've opened https://github.com/confidential-containers/confidential-containers/pull/175, which serves as the starting point for the discussion.

The content of the PR is my personal view of how this should handled and, of course, I'm more than open to adapt the text to whatever makes more sense for the project.

**Review by sameo — APPROVED** [review]

**Comment by ariel-adam** [comment]
I see no issue with adding Ryan as a 2nd representative from AMD.
Let's keep this simple with no extra process and approve this.
I'm approving it as is.

**Review by ariel-adam — APPROVED** [review]

**Review by magowan — APPROVED** [review]
I guess our charter could be tightened up but doesn't feel right to hold this up for that.

The spirit for me was we look for consensus to accept the addition of Ryan, so I am approving the PR and see if we can reach the 2/3 majority
And handle updating our charter separately..

**Review by zvonkok — APPROVED** [review]
LGTM, Ryan was very helpful in the past all things around SNP + GPU

**Review by bpradipt — APPROVED** [review]

**Review by danmihai1 — APPROVED** [review]
Sounds good to me!

**Review by vbatts — APPROVED** [review]

**Comment by fidencio** [comment]
The minimum quorum has been achieved, I'm merging it!

**Pull request merged**
